tdrs-backend/tdpservice/data_files/test/test_api.py

```diff
@@ -103,8 +103,8 @@ def assert_error_report_file_content_matches_without_friendly_names(response):
 
         assert ws.cell(row=1, column=1).value == "Error reporting in TDP is still in development.We'll" \
             + " be in touch when it's ready to use!For now please refer to the reports you receive via email"
-        assert ws.cell(row=4, column=COL_ERROR_MESSAGE).value == "if CASH_AMOUNT :873 validator1 passed" \
-            + " then NBR_MONTHS 0 is not larger than 0."
+        assert ws.cell(row=4, column=COL_ERROR_MESSAGE).value == ("if CASH_AMOUNT :873 validator1 passed then "
```

Comment:
why do we need "()"?

Reply:
It's a Python convenience tool to allow for multiline strings without having to do explicit concatenation.

Reply:
@raftmsohani, Tagging just so it doesnt get lost.